modules/library/src/main/scala/zio/elasticsearch/executor/HttpExecutor.scala

```diff
@@ -126,18 +127,26 @@ private[elasticsearch] final class HttpExecutor private (esConfig: ElasticConfig
       }
     }
 
-  private def executeBulk(r: Bulk): Task[Unit] = {
+  private def executeBulk[A](r: Bulk): Task[BulkResponse] = {
```

Comment:
Is there any usage of the type parameter `A`?

Reply:
removed in next commit.